### ISSUE TITLE
Sector First Would Restart and then Never would Commit

### ISSUE THREAD

**Issue by Meatball13** [filecoin-project/lotus#3329]
Please provide all the information requested here to help us troubleshoot "commit failed" issues.
If the information requested is missing, we will probably have to just ask you to provide it anyway,
before we can help debug.

**Describe the problem**

Tried to do a lotus-miner sectors pledge last night and found out that I had a bad/corrupt 32GiB proof cache file (more info and details listed here: https://github.com/filecoin-project/lotus/issues/3229#issuecomment-680802331).  Initially the sector would go through the entire cycle of sealing and then fail and fall back to PC1 and start over.  That was fixed by fixed the corrupt cache file, but then the sector just would never Commit.  Logs below.  Have finally just started trying to remove the sector as it was just a pledge.

A brief description of the problem you encountered while proving (sealing) a sector.

Including what commands you ran, and a description of your setup, is very helpful.
Was a clean install of 0.5.4.  Have one miner and one external worker.  Other than the sectors pledge command, didn't attempt to do anything to the sector other than stopping the miner/worker while I updated the corrupt cache file and restarted.  Had to cycle the miner a few times.

**Sectors status**

The output of `lotus-miner sectors status --log <sectorId>` for the failed sector(s).

```
lotus-miner sectors status --log 0
SectorID:	0
Status:		Committing
CIDcommD:	baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq
CIDcommR:	bagboea4b5abcaeo47w7pecvdaz753zev6zh32rdnhpxkzh2fooi7e2iaakxyjcjx
Ticket:		23c4312cf8c255ba47ecbca494d712540d871005013febca2de423a2edbffd92
TicketH:	2396
Seed:		3f765c1dd6a54f53ecd7edf1107b79cd16eacb49fad3f4d43dbb1a788ddb01e5
SeedH:		3919
Proof:		
Deals:		[0]
Retries:	1
--------
Event Log:
0.	2020-08-25 21:28:21 -0400 EDT:	[event;sealing.SectorStartCC]	{"User":{"ID":0,"SectorType":3,"Pieces":[{"Piece":{"Size":34359738368,"PieceCID":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}},"DealInfo":null}]}}
1.	2020-08-25 21:28:21 -0400 EDT:	[event;sealing.SectorPacked]	{"User":{"FillerPieces":null}}
2.	2020-08-26 00:11:29 -0400 EDT:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJyZWdpc3RlcmVkX3Byb29mIjoiU3RhY2tlZERyZzMyR2lCVjEiLCJsYWJlbHMiOnsiU3RhY2tlZERyZzMyR2lCVjEiOnsibGFiZWxzIjpbeyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTEiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9ob21lL3RidG1pbmVyL2xvdHVzU2VhbC9jYWNoZS9zLXQwODU2OC0wIiwiaWQiOiJsYXllci0yIiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItMyIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTQiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9ob21lL3RidG1pbmVyL2xvdHVzU2VhbC9jYWNoZS9zLXQwODU2OC0wIiwiaWQiOiJsYXllci01Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItNiIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTciLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9ob21lL3RidG1pbmVyL2xvdHVzU2VhbC9jYWNoZS9zLXQwODU2OC0wIiwiaWQiOiJsYXllci04Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItOSIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTEwIiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItMTEiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9XSwiX2giOm51bGx9fSwiY29uZmlnIjp7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoidHJlZS1kIiwic2l6ZSI6MjE0NzQ4MzY0Nywicm93c190b19kaXNjYXJkIjo3fSwiY29tbV9kIjpbNywxMjYsOTUsMjIyLDUzLDE5NywxMCwxNDcsMywxNjUsODAsOSwyMjcsNzMsMTM4LDc4LDE5MCwyMjMsMjQzLDE1Niw2NiwxODMsMTYsMTgzLDQ4LDIxNiwyMzYsMTIyLDE5OSwxNzUsMTY2LDYyXX0=","TicketValue":"I8QxLPjCVbpH7LyklNcSVA2HEAUBP+vKLeQjou2//ZI=","TicketEpoch":2396}}
3.	2020-08-26 01:23:50 -0400 EDT:	[event;sealing.SectorPreCommit2]	{"User":{"Sealed":{"/":"bagboea4b5abca7c47i2pyeom7cxuimgw5koa3cmcyhtlv6bjznbxnmzhavdn37ki"},"Unsealed":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}}
4.	2020-08-26 01:23:50 -0400 EDT:	[event;sealing.SectorPreCommitted]	{"User":{"Message":{"/":"bafy2bzacebhk3jwhtwmqyipdaf4xxogdk55gx4s7sqyyvv2goyhpuwdt5mphy"},"PreCommitDeposit":"1285389572403773707","PreCommitInfo":{"SealProof":3,"SectorNumber":0,"SealedCID":{"/":"bagboea4b5abca7c47i2pyeom7cxuimgw5koa3cmcyhtlv6bjznbxnmzhavdn37ki"},"SealRandEpoch":2396,"DealIDs":[],"Expiration":1556326,"ReplaceCapacity":false,"ReplaceSectorDeadline":0,"ReplaceSectorPartition":0,"ReplaceSectorNumber":0}}}
5.	2020-08-26 01:27:29 -0400 EDT:	[event;sealing.SectorPreCommitLanded]	{"User":{"TipSet":"AXGg5AIgnazAKSHqhSor7MfUoUjflKgtEQWVGBEFUv4AE5gS7eUBcaDkAiBAsqJcUfhERtkkqJrUDDBqZNJpXfoDmtwofm/R//EASAFxoOQCIKTtWOp1wykJOhk9YI2Ug48v6CN8wvArOKkUQRurqUqq"}}
6.	2020-08-26 02:42:29 -0400 EDT:	[event;sealing.SectorSeedReady]	{"User":{"SeedValue":"P3ZcHdalT1Ps1+3xEHt5zRbqy0n60/TUPbsaeI3bAeU=","SeedEpoch":3919}}
7.	2020-08-26 02:42:29 -0400 EDT:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(1): StandaloneSealCommit: Rust panic: no unwind information
8.	2020-08-26 02:43:29 -0400 EDT:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
9.	2020-08-26 02:43:29 -0400 EDT:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(1): StandaloneSealCommit: Rust panic: no unwind information
10.	2020-08-26 02:44:29 -0400 EDT:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
11.	2020-08-26 02:44:29 -0400 EDT:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(1): StandaloneSealCommit: Rust panic: no unwind information
12.	2020-08-26 02:45:29 -0400 EDT:	[event;sealing.SectorSealPreCommit1Failed]	{"User":{}}
	consecutive compute fails
13.	2020-08-26 02:46:29 -0400 EDT:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}
14.	2020-08-26 05:27:03 -0400 EDT:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJyZWdpc3RlcmVkX3Byb29mIjoiU3RhY2tlZERyZzMyR2lCVjEiLCJsYWJlbHMiOnsiU3RhY2tlZERyZzMyR2lCVjEiOnsibGFiZWxzIjpbeyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTEiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9ob21lL3RidG1pbmVyL2xvdHVzU2VhbC9jYWNoZS9zLXQwODU2OC0wIiwiaWQiOiJsYXllci0yIiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItMyIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTQiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9ob21lL3RidG1pbmVyL2xvdHVzU2VhbC9jYWNoZS9zLXQwODU2OC0wIiwiaWQiOiJsYXllci01Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItNiIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTciLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9ob21lL3RidG1pbmVyL2xvdHVzU2VhbC9jYWNoZS9zLXQwODU2OC0wIiwiaWQiOiJsYXllci04Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItOSIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTEwIiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItMTEiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9XSwiX2giOm51bGx9fSwiY29uZmlnIjp7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoidHJlZS1kIiwic2l6ZSI6MjE0NzQ4MzY0Nywicm93c190b19kaXNjYXJkIjo3fSwiY29tbV9kIjpbNywxMjYsOTUsMjIyLDUzLDE5NywxMCwxNDcsMywxNjUsODAsOSwyMjcsNzMsMTM4LDc4LDE5MCwyMjMsMjQzLDE1Niw2NiwxODMsMTYsMTgzLDQ4LDIxNiwyMzYsMTIyLDE5OSwxNzUsMTY2LDYyXX0=","TicketValue":"I8QxLPjCVbpH7LyklNcSVA2HEAUBP+vKLeQjou2//ZI=","TicketEpoch":2396}}
15.	2020-08-26 05:29:42 -0400 EDT:	[event;sealing.SectorRestart]	{"User":{}}
16.	2020-08-26 05:52:40 -0400 EDT:	[event;sealing.SectorRestart]	{"User":{}}
17.	2020-08-26 05:52:40 -0400 EDT:	[event;sealing.SectorSealPreCommit2Failed]	{"User":{}}
	seal pre commit(2) failed: failed to acquire sector {8568 0} from remote(2): sector not found
18.	2020-08-26 05:53:40 -0400 EDT:	[event;sealing.SectorRetrySealPreCommit2]	{"User":{}}
19.	2020-08-26 05:53:40 -0400 EDT:	[event;sealing.SectorSealPreCommit2Failed]	{"User":{}}
	seal pre commit(2) failed: failed to acquire sector {8568 0} from remote(2): sector not found
20.	2020-08-26 05:54:40 -0400 EDT:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}
21.	2020-08-26 06:32:45 -0400 EDT:	[event;sealing.SectorRestart]	{"User":{}}
22.	2020-08-26 06:42:34 -0400 EDT:	[event;sealing.SectorRestart]	{"User":{}}
23.	2020-08-26 11:28:17 -0400 EDT:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJyZWdpc3RlcmVkX3Byb29mIjoiU3RhY2tlZERyZzMyR2lCVjEiLCJsYWJlbHMiOnsiU3RhY2tlZERyZzMyR2lCVjEiOnsibGFiZWxzIjpbeyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTEiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9ob21lL3RidG1pbmVyL2xvdHVzU2VhbC9jYWNoZS9zLXQwODU2OC0wIiwiaWQiOiJsYXllci0yIiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItMyIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTQiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9ob21lL3RidG1pbmVyL2xvdHVzU2VhbC9jYWNoZS9zLXQwODU2OC0wIiwiaWQiOiJsYXllci01Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItNiIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTciLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9ob21lL3RidG1pbmVyL2xvdHVzU2VhbC9jYWNoZS9zLXQwODU2OC0wIiwiaWQiOiJsYXllci04Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItOSIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2hvbWUvdGJ0bWluZXIvbG90dXNTZWFsL2NhY2hlL3MtdDA4NTY4LTAiLCJpZCI6ImxheWVyLTEwIiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoibGF5ZXItMTEiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9XSwiX2giOm51bGx9fSwiY29uZmlnIjp7InBhdGgiOiIvaG9tZS90YnRtaW5lci9sb3R1c1NlYWwvY2FjaGUvcy10MDg1NjgtMCIsImlkIjoidHJlZS1kIiwic2l6ZSI6MjE0NzQ4MzY0Nywicm93c190b19kaXNjYXJkIjo3fSwiY29tbV9kIjpbNywxMjYsOTUsMjIyLDUzLDE5NywxMCwxNDcsMywxNjUsODAsOSwyMjcsNzMsMTM4LDc4LDE5MCwyMjMsMjQzLDE1Niw2NiwxODMsMTYsMTgzLDQ4LDIxNiwyMzYsMTIyLDE5OSwxNzUsMTY2LDYyXX0=","TicketValue":"I8QxLPjCVbpH7LyklNcSVA2HEAUBP+vKLeQjou2//ZI=","TicketEpoch":2396}}
24.	2020-08-26 12:40:46 -0400 EDT:	[event;sealing.SectorPreCommit2]	{"User":{"Sealed":{"/":"bagboea4b5abcaeo47w7pecvdaz753zev6zh32rdnhpxkzh2fooi7e2iaakxyjcjx"},"Unsealed":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}}
25.	2020-08-26 12:40:46 -0400 EDT:	[event;sealing.SectorPreCommitLanded]	{"User":{"TipSet":"AXGg5AIgcDIjkd0jvxGBIt5jqCx4qPxj6MgCIN6rjSdqtO9kHUQBcaDkAiBH3b5kRHQJOlelD2qvKh0eswVU+z6+1OiF2rttpHP/JgFxoOQCIJpItGwFspS8s86MU+oNf9dLFB3xF8OA8u2+I4wUPKB2AXGg5AIgM3zsLpfs/n7ujbQazBV64w4ha/9PcsmwG9m4sXf+RC8="}}
26.	2020-08-26 12:40:46 -0400 EDT:	[event;sealing.SectorSeedReady]	{"User":{"SeedValue":"P3ZcHdalT1Ps1+3xEHt5zRbqy0n60/TUPbsaeI3bAeU=","SeedEpoch":3919}}
27.	2020-08-26 16:59:00 -0400 EDT:	[event;sealing.SectorCommitFailed]	{"User":{}}
	commit check error: invalid proof (compute error?)
28.	2020-08-26 17:00:00 -0400 EDT:	[event;sealing.SectorRetryInvalidProof]	{"User":{}}
```

**Lotus miner logs**

Please go through the logs of your miner, and include screenshots of any error-like messages you find.

`grep 8568-0 miner.log
2020-08-26T06:42:34.950-0400	DEBUG	advmgr	sector-storage/localworker.go:75	acquired sector {8568 0} (e:1; a:0): {{0 0} /home/tbtminer/lotusSeal/unsealed/s-t08568-0  }
2020-08-26T06:42:34.950-0400	INFO	stores	stores/local.go:514	remove /home/tbtminer/lotusSeal/sealed/s-t08568-0
2020-08-26T06:42:36.048-0400	INFO	stores	stores/local.go:514	remove /home/tbtminer/lotusSeal/cache/s-t08568-0
2020-08-26T06:42:38.485-0400	DEBUG	advmgr	sector-storage/localworker.go:75	acquired sector {8568 0} (e:1; a:6): {{0 0} /home/tbtminer/lotusSeal/unsealed/s-t08568-0 /home/tbtminer/lotusSeal/sealed/s-t08568-0 /home/tbtminer/lotusSeal/cache/s-t08568-0}
2020-08-26T11:28:17.723-0400	DEBUG	advmgr	sector-storage/localworker.go:75	acquired sector {8568 0} (e:6; a:0): {{0 0}  /home/tbtminer/lotusSeal/sealed/s-t08568-0 /home/tbtminer/lotusSeal/cache/s-t08568-0}
2020-08-26T11:28:17.724-0400	DEBUG	advmgr	sector-storage/localworker.go:75	acquired sector {8568 0} (e:6; a:0): {{0 0}  /home/tbtminer/lotusSeal/sealed/s-t08568-0 /home/tbtminer/lotusSeal/cache/s-t08568-0}
2020-08-26T12:40:46.233 INFO storage_proofs_core::data > dropping data /home/tbtminer/lotusSeal/sealed/s-t08568-0
2020-08-26T12:40:46.750-0400	DEBUG	advmgr	sector-storage/localworker.go:75	acquired sector {8568 0} (e:6; a:0): {{0 0}  /home/tbtminer/lotusSeal/sealed/s-t08568-0 /home/tbtminer/lotusSeal/cache/s-t08568-0}
2020-08-26T12:40:46.750-0400	DEBUG	advmgr	sector-storage/localworker.go:75	acquired sector {8568 0} (e:6; a:0): {{0 0}  /home/tbtminer/lotusSeal/sealed/s-t08568-0 /home/tbtminer/lotusSeal/cache/s-t08568-0}
2020-08-26T17:00:00.020-0400	DEBUG	advmgr	sector-storage/localworker.go:75	acquired sector {8568 0} (e:6; a:0): {{0 0}  /home/tbtminer/lotusSeal/sealed/s-t08568-0 /home/tbtminer/lotusSeal/cache/s-t08568-0}
2020-08-26T17:00:00.020-0400	DEBUG	advmgr	sector-storage/localworker.go:75	acquired sector {8568 0} (e:6; a:0): {{0 0}  /home/tbtminer/lotusSeal/sealed/s-t08568-0 /home/tbtminer/lotusSeal/cache/s-t08568-0}
`

```
grep 8568-0 worker.log
2020-08-26T05:31:11.503-0400	INFO	stores	stores/remote.go:221	Fetch http://172.16.0.1:3456/remote/sealed/s-t08568-0 -> /home/tbtminer/worker/sealed/fetching/s-t08568-0
2020-08-26T05:37:46.384-0400	DEBUG	stores	stores/util_unix.go:28	move sector data	{"from": "/home/tbtminer/worker/sealed/fetching/s-t08568-0", "to": "/home/tbtminer/worker/sealed/s-t08568-0"}
2020-08-26T05:37:46.394-0400	INFO	stores	stores/remote.go:322	Delete http://172.16.0.1:3456/remote/sealed/s-t08568-0
2020-08-26T05:37:46.396-0400	INFO	stores	stores/remote.go:221	Fetch http://172.16.0.1:3456/remote/cache/s-t08568-0 -> /home/tbtminer/worker/cache/fetching/s-t08568-0
2020-08-26T05:44:56.681-0400	WARN	rpc	go-jsonrpc@v0.1.2-0.20200822201400-474f4fdccc52/handler.go:241	error in RPC call to 'Filecoin.Fetch': failed to acquire sector {8568 0} from remote (tried [{78c52f61-8fb7-40b9-9782-74db3a069772 [http://172.16.0.1:3456/remote/cache/s-t08568-0] 10 true false false}]):
	* fetch error http://172.16.0.1:3456/remote/cache/s-t08568-0 (storage 78c52f61-8fb7-40b9-9782-74db3a069772) -> /home/tbtminer/worker/cache/fetching/s-t08568-0: context canceled
2020-08-26T05:54:40.017-0400	INFO	stores	stores/http_handler.go:120	SERVE DELETE /remote/sealed/s-t08568-0
2020-08-26T05:54:40.021-0400	INFO	stores	stores/local.go:514	remove /home/tbtminer/worker/sealed/s-t08568-0

```

Alternatively please upload full log files and share a link here

**Lotus miner diagnostic info**

Please collect the following diagnostic information, and share a link here

* lotus-miner diagnostic info `lotus-miner info all > allinfo`

** Code modifications **

If you have modified parts of lotus, please describe which areas were modified,
and the scope of those modifications

**Version**

The output of `lotus --version`.

lotus version
Daemon:  0.5.4+git.d4fef1b5+api0.12.0
Local: lotus version 0.5.4+git.d4fef1b5

**Comment by Reiers** [comment]
Hi @Meatball13 

Thanks for the report. This is fixed in the latest release.
Please upgrade Lotus.

*Link to latest tags [here.](https://github.com/filecoin-project/lotus/tags)*

Closing ticket.

If you are still experiencing problems, open a new ticket or ask for help in our [Discussions forum.](https://github.com/filecoin-project/lotus/discussions)
